curriculum/challenges/english/25-front-end-development/lab-email-masker/66b205e6eacba4c4e54ea434.md

```diff
@@ -0,0 +1,118 @@
+---
+id: 66b205e6eacba4c4e54ea434
+title: Build an Email Masker
+challengeType: 14
+dashedName: lab-email-masker
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. Declare a variable named `email` to store the email address you want to mask. 
+2. Split the given email address by `@` using `slice()` to get the username and domain name. 
+3. Store the username and domain name in `userName` and `domain` variables respectively.
+4. The first letter and last letter of the username shouldn't change. Replace the remaining characters with asterisks (*) to mask it. Store the masked username in a variable named `maskedName`.
+5. Combine the `maskedName` and `domain` to get the masked email address and store it in a variable named `finalMaskedEmail`.
+6. Print the `finalMaskedEmail` to the console.
+
+# --hints--
```

Comment:
we should update the tests to match the outlines suggestions I mentioned earlier.